test/env.js

```diff
@@ -16,3 +16,4 @@ process.env.PUSH_NOTIFICATIONS_SERVICE_URL =
 process.env.PORTFOLIO_URL = 'https://portfolio.test';
 process.env.METAMASK_VERSION = 'MOCK_VERSION';
 process.env.ENABLE_CONFIRMATION_REDESIGN = 'true';
+process.env.TZ = 'UTC';
```

Comment:
Unsure if this actually has an impact in tests. Can I get other devs to test with/without this environment var?

Reply:
I will keep this in the test environment for now. It does not impact any other tests.

Leaving a comment here for future devs: This can be taken out if necessary 😄